package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@show-runner/fixturelibrary",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 
> MAJOR version when you make incompatible API changes
> MINOR version when you add functionality in a backwards compatible manner
> PATCH version when you make backwards compatible bug fixes
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

[Spec](https://semver.org/)

```suggestion
  "version": "2.2.0",
```

I think this is a feature, not bug fix.